packages/ui/components/cards/UpcomingChainCard.vue

```diff
@@ -1,12 +1,14 @@
 <template>
-  <a :href="path" target="_blank">
-    <div class="card-container">
-      <div class="title">
-        <SvgIcon class="img" :name="img" />
-        <p class="name title">{{ name }}</p>
+  <BaseCard>
```

Comment:
Do the upcoming chains need to be Card? Do they have border, background or any other Card property?